src/Microsoft.DotNet.CodeAnalysis/PackageFiles/codeAnalysis.props

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <ItemGroup Condition="'$(EnableDotnetAnalyzers)' == 'true'">
+    <!-- Enable the Microsoft.Dotnet.CodeAnalysis.dll analyzer --> 
+    <Analyzer Include="$(BuildToolsTaskDir)Microsoft.Dotnet.CodeAnalysis.dll"/>
```

Comment:
While it is fine this is a props file for now I suspect you may end up needing a targets file at some point so if you think that is now then it might be worth going ahead and doing a targets file instead.